Misc/NEWS.d/next/Library/2019-05-28-02-37-00.bpo-36520.W4tday.rst

```diff
@@ -0,0 +1 @@
+Lengthy email headers with UTF-8 characters are now properly encoded when they are folded.
```

Comment:
Can you also add `Path by <your name>` to the end for correct attribution in changelog?

Reply:
I added my name to the NEWS blurb.